Correct layout mistake

JipFr opened the pull request:
Comments with long strings (such as URLs) would previously break the layout. With the fixed word break, this is now corrected

matteing reviewed (commented):
Not sure if this is a good idea. Tailwind already has primitives for break-word, adding a class isn't necessary.

JipFr commented:
I believe the `break-words` class does not apply this rule, instead it applies a rule that does not actually break the lines. If you look at any reply with a long URL you will see neither `break-words` nor `break-word` correct the issue, so I added this.

I was also unable to find the relevant field on the Tailwind documentation, so I added this.

matteing commented:
Take it back, just searched the docs, it uses overflow-wrap vs. word-wrap. Not sure what the difference is, but I'll accept the commit.

Merges #2
